applications/openshift/kubelet/kubelet_disable_hostname_override/rule.yml

```diff
@@ -2,29 +2,25 @@ documentation_complete: true
 
 prodtype: ocp4
 
-title: 'kubelet - Disable Hostname Override'
+title: 'kubelet - Hostname Override handling'
 
 description: |-
-    To prevent the hostname from being overrided, edit the kubelet systemd service file
-    <tt>/etc/systemd/system/kubelet.service</tt> on the kubelet node(s) and
-    remove the <tt>--hostname-override</tt> flag if it exists.
+    Normally, OpenShift lets the kubelet get the hostname from either the
+    cloud provider itself, or from the node's hostname. This ensures that
+    the PKI allocated by the deployment uses the appropriate values, is valid
+    and keeps working throughout the lifecycle of the cluster. IP addresses
+    are not used, and hence this makes it easier for security analysts to
+    associate kubelet logs with the appropriate node.
 
 rationale: |-
-    Allowing hostnames to be overrided creates issues around resolving nodes
+    Allowing hostnames to be overridden creates issues around resolving nodes
     in addition to TLS configuration, certificate validation, and log correlation
-    and validation.
+    and validation. However, in some cases explicit overriding of this parameter
```

Comment:
(nit) "overriding of this parameter **is** necessary".

Reply:
aside from this, LGTM